cmake/CMakeLists.txt

```diff
@@ -371,7 +374,7 @@ configure_file(
 add_definitions(-DHAVE_CONFIG_H)
 
 install(
-    TARGETS ${LIBRARY_STATIC} ${LIBRARY_SHARED} ${OSCDUMP} ${OSCSEND}
+	TARGETS ${LIBRARY_STATIC} ${LIBRARY_SHARED} ${OSCDUMP} ${OSCSEND} ${OSCSENDFILE}
```

Comment:
Spaces replaced by tabs. You should probably keep tabs-vs-spaces consistent.

Reply:
Thanks for the review. Pushed a commit to fix these issues.